lib/util.js

```diff
@@ -26,15 +26,11 @@ function * ensureNonStarterPlan (db) {
 }
 
 function * newTotalExecTimeField (db) {
-  const newTotalExecTimeFieldQuery = 'SELECT current_setting(\'server_version_num\')::numeric >= 130000'
-  const newTotalExecTimeFieldRaw = yield pg.psql.exec(db, newTotalExecTimeFieldQuery)
+  let newTotalExecTimeFieldQuery = `SELECT current_setting('server_version_num')::numeric >= 130000`
+  let newTotalExecTimeFieldRaw = yield pg.psql.exec(db, newTotalExecTimeFieldQuery, ['-t', '-q'])
 
   // error checks
-  let newTotalExecTimeField = newTotalExecTimeFieldRaw.split('\n')
-  if (newTotalExecTimeField.length !== 6) {
-    throw new Error('Unable to determine database version')
-  }
-  newTotalExecTimeField = newTotalExecTimeFieldRaw.split('\n')[2].trim()
+  newTotalExecTimeField = newTotalExecTimeFieldRaw.split("\n")[0].trim()
```

Comment:
This hasn't been previously defined so also needs a declaration:
```suggestion
  const newTotalExecTimeField = newTotalExecTimeFieldRaw.split("\n")[0].trim()
```